### ISSUE TITLE
When switching between files, the Outline panel search input steals focus

### ISSUE THREAD

**Issue by mitchellnemitz** [facebook-atom/atom-ide-ui#105]
### Before Submitting
- [x] Have you installed an "ide-" language package? atom-ide-ui provides no functionality
  out-of-the box unless you've installed a suitable package.
- [x] If your issue is language-specific, please file an issue against
  the "ide-" language package you're using instead.

### Description / Steps to Reproduce
**Note:** With MRU file switching disabled and classic left-to-right `ctrl-tab` behavior.

When using `ctrl-tab` to switch to a file where the Outline panel is visible and a language server is installed, focus is given to the "Search Outline" input at the top of the Outline panel instead of the editor.

The first case with which I discovered this issue was switching from a yaml file without `ide-yaml` installed (so the Outline panel is empty) to a PHP file with `ide-php` installed (where the Outline panel has a search input available).

Admittedly a small bug, but an incredibly frustrating one in practice.

### Expected Behavior
The core editor should obtain focus with the cursor placed at its last known position.

### Actual Behavior
The focus is taken by the "Search Outline" input at the top of the Outline panel.

### Versions
* Atom: 1.21.1
* Client OS: macOS Sierra Version 10.12.6 (16G29)
* atom-ide-ui: 0.5.3

### Additional Details
```
$ apm ls --installed
Community Packages (29) /Users/mitchell/.atom/packages
├── atom-beautify@0.30.5
├── atom-ide-ui@0.5.3
├── busy-signal@1.4.3
├── ex-mode@0.18.0
├── file-icons@2.1.13
├── hey-pane@1.0.0
├── ide-css@0.2.0
├── ide-html@0.3.0
├── ide-json@0.1.0
├── ide-php@0.6.9
├── ide-typescript@0.6.2
├── intentions@1.1.5
├── language-docker@1.1.8
├── language-gherkin@1.0.4
├── language-ini@1.19.0
├── linter@2.2.0
├── linter-docker@0.2.1
├── linter-jshint@3.1.6
├── linter-phpcs@1.6.8
├── linter-ui-default@1.6.10
├── minimap@4.29.7
├── minimap-git-diff@4.3.1
├── monokai@0.24.0
├── native-ui@0.24.0
├── php-debug@0.2.5
├── php-twig@4.0.0
├── platformio-ide-terminal@2.7.0
├── single-click-open@0.2.2
└── vim-mode-plus@1.12.1
```

**Comment by hansonw** [comment]
Ooh this should've been fixed in https://github.com/facebook-atom/atom-ide-ui/commit/8140bb2a8cec1a011975d09e0ccaf0d2c6b1efc6, but it looks like it doesn't cover all the cases. We'll take a look.

**Comment by mitchellnemitz** [comment]
Wow, that was fast. Thanks, @hansonw!

**Comment by lloiser** [comment]
It always happens for me when I switch from a file that has no outline provider (e.g. `.md` files) to a file that has an outline provider (e.g. `.js`)

**Comment by hansonw** [comment]
Fixed in 081327d4a97e798c6aa0efcf92074d475f624dd5 - I'll be publishing a patch release sometime this week.
